src/main/java/com/minecolonies/coremod/entity/ai/citizen/lumberjack/EntityAIWorkLumberjack.java

```diff
@@ -104,7 +103,7 @@
     /**
      * Time in ticks to wait before rechecking if there are trees in the range of the lumberjack.
      */
-    private static final int WAIT_BEFORE_SEARCH = 100;
+    private static final int WAIT_BEFORE_SEARCH = 400;
```

Comment:
problem with restriction area is that when there is no trea + the area is far then the range attribute is big so the search is big, this probably needs a bit more investigation on how to solve that properly, so making the delay 20 secs